index.html

```diff
@@ -3,11 +3,187 @@
+    </header> 
+       
+        <!-- HERO SECTION -->
+    <section class="hero">
+        <div class="h-content">
+            <div class="main-content">
+                <p>"Hello, World"</p>
+                <h1>PAKISTAN'S POLITICS 
+                    <br>
+                    FROM 1947 TO 2023.
+                <h1>
+            </div>
+            <div class="content-detail">
+                <P>Pakistan gained independence from British colonial rule in 1947 after the partition of India. It has since faced political and social challenges, but has made progress in areas such as education and infrastructure development.</P>
+            </div>
+            <p class="date">2023.03.14(Tue) ~ 16(Fri)</p>
+            <p class="address">@ Nation Museum of Korea, Art Center Nabi and more</p>
+        </div>
+    </section>
+
+        <!-- TIME PERIODS -->
+    <section class="time-period">
+        <div class="main-heading">
+            <p>Time Periods</p>
+            <div class="line"></div>
+        </div>
+        <div class="all-boxes">
+            <div class="box">
+                <img src="./images/img/Icon-1.svg">
+                <p class="date-col">1958</p>
+                <p class="desc-style">Pakistan's early years were unstable, led by Muslim League's Jinnah until his death. Weak civilian govts followed until 1958.</p>
+            </div>
+            <div class="box">
+                <img src="./images/img/Icon-1.svg">
+                <p class="date-col">1971</p>
+                <p class="desc-style">Ayub Khan's coup in 1958 granted him vast powers, dissolved the National Assembly. War with India in 1965.</p>
+            </div>
+            <div class="box">
+                <img src="./images/img/Icon-1.svg">
+                <p class="date-col">1977</p>
+                <p class="desc-style">Bangladesh declared independence in 1971, won civil war, became independent. 1973 constitution established Pakistan as republic.</p>
+            </div>
+            <div class="box">
+                <img src="./images/img/Icon-1.svg">
+                <p class="date-col">1988</p>
+                <p class="desc-style">Zia-ul-Haq took power in 1977, imposed martial law and Islamic laws, curtailed civil liberties. He died in 1988.</p>
+            </div>
+            <div class="box">
+                <img src="./images/img/Icon-1.svg">
+                <p class="date-col">2008</p>
+                <p class="desc-style">Musharraf's rule saw economic and social reforms, better ties with West, but marred by corruption, abuses. Declared emergency in 2007, resigned in 2008.</p>
+            </div>
+        </div>
+        <a class="detail-link">See the whole program</a>
+    </section>
+
+        <!-- RULER'S SECTION -->
+    <section class="pak-rulers">
+        <div class="main-heading r-heading">
+            <p>Pakistan's Rulers</p>
+            <div class="line"></div>
+        </div>
+        <div class="all-leaders">
```

Comment:
Please follow the original design see 👇

![ruler2](https://github.com/ibtisam-adil/Capstone/assets/118625824/45a6ccf5-298e-4c8c-a329-7d00e4c5b4d2)

how it should be 👇
![rulers-1](https://github.com/ibtisam-adil/Capstone/assets/118625824/85c1ef89-46e3-4d9c-9f1f-2cb7f6a876eb)